src/components/HomePage/SearchResultsArea/SearchResultsWidget/OfferItemsContainer.js

```diff
@@ -17,6 +17,8 @@ import clsx from "clsx";
 
 const useAdvancedSearchButtonStyles = makeStyles((theme) => ({
     root: {
+        top: "0",
+        position: "sticky",
```

Comment:
```suggestion
        top: 0,
        position: "sticky",
        zIndex: 1,
```

You can solve the problem discussed in https://github.com/NIAEFEUP/nijobs-fe/issues/186#issuecomment-1059385124 by adding the `zIndex: 1,` line, which makes the button be in the front of the offers.

However, make sure the offers behind the button are not visible.

![1](https://user-images.githubusercontent.com/50210213/156832225-ffe0f4af-169f-4b80-a0f9-371058fae4e4.png)

Reply:
Wow, it was that simple.
Regarding the last sentence, what you're suggesting is that I hide the offers dynamically as they go past the filters button?

Reply:
This is the current behaviour of the button: It hides parts of the offer that should be behind it.
![2022-03-04-21:26:13-screenshot](https://user-images.githubusercontent.com/75742355/156844079-81bf4b6a-63f5-47a8-833e-9337ddae339e.png)

Reply:
If you don't want the button to stay on top of the offers, you don't need to hide them. You could simply put the button outside of the scroll area

Reply:
The offers should not show behind the button, it should have a "clear area" around it. It looks broken this way

Reply:
After searching for hours trying to come up with a solution involving margins and padding (without success), I found out that adding `backgroundColor: "white"` colored the button header and now hides the offers behind it on scroll. Is this an acceptable solution?

```suggestion
        top: "0",
        position: "sticky",
        zIndex: 1,
        backgroundColor: "white",
```
Here's how it looks:

![2022-03-13-14:29:04-screenshot](https://user-images.githubusercontent.com/75742355/158064336-e86ecb15-4080-4204-a896-329dfcb37e9a.png)

I also increased the padding at the bottom of the header to match the padding on top: It looks better with the header changes, in my opinion. What do you think?

Reply:
At first sight, I would say it's acceptable, but please commit the changes to give better feedback.

Reply:
Done. Sorry for taking so long, lack of experience made me overcomplicate (I think).